driver/razerkbd_driver.c

```diff
@@ -612,6 +612,130 @@ static ssize_t razer_attr_write_mode_macro_effect(struct device *dev, struct dev
+    case USB_DEVICE_ID_RAZER_BLADE_EARLY_2020_BASE:
+    default:
+        if (count < 1) printk(KERN_WARNING "razerkbd: fan speed expects 1 bytes: fan_speed");
+        report = razer_chroma_set_fan_speed(fan_id, buf[1]);
+        break;
+    }
+    response = razer_send_payload(usb_dev, &report);
+    return count;
+}
+static ssize_t razer_attr_write_fan_speed_1(struct device *dev, struct device_attribute *attr, const char *buf, size_t count)
+{
+    return razer_attr_write_fan_speed(dev,attr,buf,count,0x01);
+}
+static ssize_t razer_attr_write_fan_speed_2(struct device *dev, struct device_attribute *attr, const char *buf, size_t count)
+{
+    return razer_attr_write_fan_speed(dev,attr,buf,count, 0x02);
+}
+/**
+ * Read device file "fan_mode"
+ * Returns  fan_mode, game_mode
+ */
+static ssize_t razer_attr_read_fan_mode(struct device *dev, struct device_attribute *attr, char *buf, unsigned char fan_id)
+{
+    struct usb_interface *intf = to_usb_interface(dev->parent);
+    struct usb_device *usb_dev = interface_to_usbdev(intf);
+    struct razer_report report;
+    struct razer_report response;
+
+
+    switch(usb_dev->descriptor.idProduct) {
+    case USB_DEVICE_ID_RAZER_BLADE_EARLY_2020_BASE:
+    default:
+        report = razer_chroma_get_fan_mode(fan_id);
+        response = razer_send_payload(usb_dev, &report);
+        return sprintf(buf, "%d%d\n", response.arguments[3],response.arguments[2]);
+    }
+}
+static ssize_t razer_attr_read_fan_mode_1(struct device *dev, struct device_attribute *attr, char *buf)
+{
+    return razer_attr_read_fan_mode(dev,attr,buf,0x01);
+}
+static ssize_t razer_attr_read_fan_mode_2(struct device *dev, struct device_attribute *attr, char *buf)
+{
+    return razer_attr_read_fan_mode(dev,attr,buf, 0x02);
+}
+
+/**
+ * Write device file "fan_mode"
+ *
+ * Returns a string
+ */
+static ssize_t razer_attr_write_fan_mode(struct device *dev, struct device_attribute *attr, const char *buf, size_t count, unsigned char fan_id)
+{
+    struct usb_interface *intf = to_usb_interface(dev->parent);
+    struct usb_device *usb_dev = interface_to_usbdev(intf);
+    struct razer_report report;
+    struct razer_report response;
+
+    switch(usb_dev->descriptor.idProduct) {
+    case USB_DEVICE_ID_RAZER_BLADE_EARLY_2020_BASE:
+    default:
+        if (count < 2) printk(KERN_WARNING "razerkbd: fan mode expects 2 bytes: fan_mode, game_mode");
+        report = razer_chroma_set_fan_mode(fan_id, buf[0], buf[1]);
+        break;
+    }
+    response = razer_send_payload(usb_dev, &report);
+    return count;
+}
+static ssize_t razer_attr_write_fan_mode_1(struct device *dev, struct device_attribute *attr, const char *buf, size_t count)
+{
+    return razer_attr_write_fan_mode(dev,attr,buf,count,0x01);
+}
+static ssize_t razer_attr_write_fan_mode_2(struct device *dev, struct device_attribute *attr, const char *buf, size_t count)
+{
+    return razer_attr_write_fan_mode(dev,attr,buf,count, 0x02);
```

Comment:
add some spaces between the argument (no clue why the auto-formatter doesn't catch this...)

same for the other functions, like `razer_attr_read_fan_mode_1` and `razer_attr_write_fan_mode_1`